src/commands/publish/route.rs

```diff
@@ -1,129 +1,124 @@
-use crate::http;
-use crate::settings::global_user::GlobalUser;
-use crate::settings::toml::Target;
-use crate::terminal::emoji;
-use reqwest::header::CONTENT_TYPE;
-use serde::{Deserialize, Serialize};
+use serde::Serialize;
 
-use log::info;
+use cloudflare::endpoints::workers::{CreateRoute, CreateRouteParams, ListRoutes};
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::HttpApiClientConfig;
 
-#[derive(Deserialize, Serialize)]
-pub struct Route {
-    script: Option<String>,
-    pub pattern: String,
+use crate::http::{api_client, format_error};
+use crate::settings::global_user::GlobalUser;
+use crate::settings::toml::Route;
+
+pub fn publish_routes(
+    user: &GlobalUser,
+    routes: Vec<Route>,
+    zone_id: &String,
```

Comment:
I think we should be accepting an &str here https://stackoverflow.com/questions/40006219/why-is-it-discouraged-to-accept-a-reference-to-a-string-string-vec-vec-o